modules/infra/submodules/storage/netapp.tf

```diff
@@ -44,15 +46,78 @@ resource "random_password" "netapp" {
   min_lower   = 1
 }
 
+
+resource "null_resource" "secrets_cleanup" {
```

Comment:
we might want to use `terraform_data` since `null_resource` is deprecated in tf >= `1.4` 

https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource

https://developer.hashicorp.com/terraform/language/resources/terraform-data